src/profile-logic/process-profile.js

```diff
@@ -733,9 +739,6 @@ function _adjustMarkerTimestamps(
       if (typeof newData.endTime === 'number') {
         newData.endTime += delta;
       }
-      if (newData.type === 'DOMEvent' && 'timeStamp' in newData) {
-        newData.timeStamp += delta;
-      }
```

Comment:
we don't have this data anymore anywhere ?

Reply:
Oh, actually we do. Adding back with appropriate changes, thanks.